### PR TITLE
Add tap

### PR DESCRIPTION
**Pull request by outscale-mgo**

### CODE REVIEW

File: tests/functions.sh [hunk 1]
```diff
@@ -584,7 +584,7 @@ function server_start_options {
     options=${@:2}
     echo "[butterfly-$id] starting"
 
-    exec sudo $BUTTERFLY_BUILD_ROOT/api/server/butterflyd --dpdk-args "--no-shconf -c1 -n1 --vdev=eth_pcap$id,iface=but$id --no-huge" -l debug -i ::101 -s /tmp --endpoint=tcp://0.0.0.0:876$id $options -t &> $BUTTERFLY_BUILD_ROOT/butterflyd_${id}_output &
```
Comment (outscale-toa) [inline tests/functions.sh]: can you remove the whitespace please?

File: tests/scenario_48/test.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```
Comment (outscale-toa) [inline tests/scenario_48/test.sh]: can you add a README.d for this test please?

File: tests/scenario_49/test.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
```
Comment (outscale-toa) [inline tests/scenario_49/test.sh]: can you add a README.d for this test please?

File: tests/scenario_50/test.sh [hunk 4]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
```
Comment (outscale-toa) [inline tests/scenario_50/test.sh]: can you add a README.d for this test please?

**Comment by outscale-mgo** [comment]
merged

**Pull request closed without merging**